plugins/colorbutton/plugin.js

```diff
@@ -154,6 +154,23 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					attributes: { role: 'listbox', 'aria-label': lang.panelTitle }
 				},
 
+				select: function( callback ) {
+					var colors = config.colorButton_colors.split( ',' ),
+						colorToSelect;
+
+					for ( var i = 0; i < colors.length; i++ ) {
```

Comment:
We have `CKEDITOR.tools.array#find()` method – it should increase the readability of this code.

In fact it would allow to pass `callback` directly to `colors`:

```javascript
var colorToSelect = normalizeColor( CKEDITOR.tools.array.find( colors, callback ) );
```

Reply:
There was something wrong with this code but I couldn't catch up what (had some brain fart yesterday). Obviously, a find method, thanks :smile: